RELEASES.md

```diff
@@ -7,6 +7,14 @@ Releases are proposed and adopted or rejected using the usual [project governanc
 An anti-pattern that we want to avoid is heavy development or discussions "late cycle" around major releases.
 We want to build a community that is involved and communicates consistently through all releases instead of relying on "silent periods" as a judge of stability.
 
+## Release Process
```

Comment:
> If its not runtime-spec specific then it shouldn't be in this repo at all. I'm inclined to say we remove it entirely and put it at a higher level in the OCI git repo structure, but let's do that in a separate PR.

The release-doc issue seems orthogonal to the README decoupling, so I'd rather not touch them in this PR.

And the local copy is because each project can tweak the generic docs, although they're [encouraged not to][1].

[1]: https://github.com/opencontainers/project-template/issues/4#issuecomment-221081999